### PR TITLE
Edit grammar and formatting of quickstart article

### PR DESCRIPTION
**Pull request by myoungerman**
I'm the new technical writer! This is my first PR for Veritone's docs, I've made some minor formatting and grammar changes to an article.

### CODE REVIEW

**Review by faryeyay — APPROVED** [review]
LGTM

File: src/docs/getting-started/quickstart/README.md [hunk 1]
```diff
@@ -8,73 +8,59 @@ th { text-align:left; }
...
 
-> Visit [automate.veritone.com](https://automate.veritone.com/)
-
-Veritone **Automate Studio** is a low-code workflow designer that empowers technical and business teams to tap into a full-stack AI architecture to design and deploy AI-powered business processes at scale in days, not months. Leverage an intuitive drag-and-drop UI to easily create advanced business logic on a digital canvas, without the need for in-depth coding skills or AI expertise. Automate Studio enables intelligent process automation (IPA) and workflow enrichment across both new and existing systems and applications.
-
-?>Learn how to use Automate Studio and build your first flow by visiting the [Automate Studio](/automate-studio/) section
+    ?>Learn how to use Automate Studio and build your first flow by visiting the [Automate Studio](/automate-studio/) section.
```
Comment (hm90121) [inline src/docs/getting-started/quickstart/README.md:63]: can u omit the ?> it's showing in the deployment

File: src/docs/getting-started/quickstart/README.md [hunk 2]
```diff
@@ -8,73 +8,59 @@ th { text-align:left; }
 
 **APPROXIMATE READING TIME: 8 MINUTES**
 
-It is easy to get started with Veritone's aiWARE platform. Running a cognition job is just few steps away.
+It's easy to get started with Veritone's aiWARE platform. Running a cognition job is just a few steps away.
 
 
 ## Step 1: Register for free <!-- {docsify-ignore} -->
```
Comment (hm90121) [inline src/docs/getting-started/quickstart/README.md:14]: can u provide screenshots for the below steps

File: src/docs/getting-started/quickstart/README.md [hunk 3]
```diff
@@ -85,34 +71,30 @@ Veritone **Automate Studio** is a low-code workflow designer that empowers techn
...
 
 <div class="collapse-accordion"><ul><li>
                 <input type="checkbox" id="list-item-3">
-                <label for="list-item-3"><span class="expandText">Click here to learn run your first engine</span><span class="collapseText">Click here to close this section.</span></label>
+                <label for="list-item-3"><span class="expandText">Learn how to run your first engine</span><span class="collapseText">Click here to close this section.</span></label>
                 <ul>
                     <li class="inner-content">
                     
 
-1. From the Apps menu select the Automate Studio app. It will open in a new tab.
```
Comment (hm90121) [inline src/docs/getting-started/quickstart/README.md:97]: can u provide screenshots for every step

File: src/docs/getting-started/quickstart/README.md (hunk 1 above)
Comment (myoungerman) [reply src/docs/getting-started/quickstart/README.md]: @ hm90121 Sure, I've removed it in the latest commit.

File: src/docs/getting-started/quickstart/README.md (hunk 3 above)
Comment (myoungerman) [reply src/docs/getting-started/quickstart/README.md:97]: @hm90121 This is my 3rd day, so I don't know yet where I go to find screenshots :) I'm definitely open to adding some once I know where to look, but my goal right now is to improve the grammar and formatting.

**Pull request merged**